core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/ColumnsSelectionDsl.kt

```diff
@@ -3,181 +3,159 @@ package org.jetbrains.kotlinx.dataframe.api
 import io.kotest.matchers.shouldBe
 import org.jetbrains.kotlinx.dataframe.DataFrame
 import org.jetbrains.kotlinx.dataframe.annotations.DataSchema
-import org.jetbrains.kotlinx.dataframe.impl.columns.asValueColumn
-import org.jetbrains.kotlinx.dataframe.samples.api.TestBase
-import org.jetbrains.kotlinx.dataframe.samples.api.age
-import org.jetbrains.kotlinx.dataframe.samples.api.firstName
-import org.jetbrains.kotlinx.dataframe.samples.api.isHappy
-import org.jetbrains.kotlinx.dataframe.samples.api.lastName
-import org.jetbrains.kotlinx.dataframe.samples.api.name
+import org.jetbrains.kotlinx.dataframe.samples.api.*
 import org.junit.Test
+import kotlin.reflect.typeOf
 
-class ColumnsSelectionDslTests : TestBase() {
+open class ColumnsSelectionDslTests : TestBase() {
 
     @Test
     fun first() {
-        df.select { all().first() } shouldBe df.select { first() }
-        df.select { all().first() } shouldBe df.select { name }
-        df.select { first() } shouldBe df.select { name }
-        df.select { first { it.name().startsWith("a") } } shouldBe df.select { age }
-
-        df.select {
-            name.first { col ->
-                col.any { it == "Alice" }
-            }
-        } shouldBe df.select {
-            name.colsOf<String>().first { col ->
-                col.any { it == "Alice" }
-            }
-        }
+        listOf(
+            df.select { name },
+            df.select { first() },
+            df.select { all().first() },
+            df.select { first { it.name().startsWith("n") } },
+        ).shouldAllBeEqual()
```

Comment:
Should they be equal between each other? Or to another snippet of API

I mean, that for now, the test structure is hidden a little bit and naming could lead to the misunderstanding

But the goal of refactoring is clear

Reply:
Yes, they should all be equal to each other. I think the same is clear. It cannot mean anything else, right?